resources/assetstore/charts/asset-store-controller-manager/templates/statefulset.yaml

```diff
@@ -45,10 +45,7 @@ spec:
                 name: assetstore-minio-docs-upload
                 key: APP_UPLOAD_ENDPOINT_WITH_PORT
           - name: APP_STORE_EXTERNAL_ENDPOINT
-            valueFrom:
-              configMapKeyRef:
-                name: assetstore-minio-docs-upload
-                key: APP_EXTERNAL_ENDPOINT
+            value: {{ if .Values.global.minioExternalEndpoint }}{{ .Values.global.minioExternalEndpoint }}{{ else }}'https://minio.{{ .Values.global.ingress.domainName }}'{{ end }}
```

Comment:
If you use global variables in this condition and in various places in asset-store charts, maybe would be better to extract this logic to global values as storeExternalEndpoint?

Reply:
`values.yaml` may contain only values, logic is not supported.